cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -86,14 +88,15 @@ def _create_content_experiment(self, cid=-1, name_suffix='', special_characters=
         self.save_course()
         return (vertical, split_test)
 
-    def _create_problem_with_content_group(self, cid, group_id, name_suffix='', special_characters=''):
+    def _create_problem_with_content_group(self, cid, group_id, course=None, name_suffix='', special_characters=''):
```

Comment:
If you're worried about the cost of creating a course twice, you can pass a kwarg here to prevent the setUp from creating a course: https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/tests/utils.py#L75-L89